rpm/gen-fixtures-delta.sh

```diff
@@ -0,0 +1,160 @@
+#!/usr/bin/env sh
+#
+# Generate RPM repository with DRPM fixture data. Several RPM utilities must be installed,
+# including `createrepo` and `makedeltarpm`.
+#
+set -euo pipefail
+
+#------------------------------------------------------------------------------
+# Helper Functions
```

Comment:
Nitpick: either drop this headline, or add a "business logic" headline below. I'd advocate for the former.